src/pybind/mgr/dashboard/frontend/src/app/core/navigation/notifications/notifications.component.html

```diff
@@ -47,7 +47,8 @@
 <a [popover]="popTemplate"
    placement="bottom"
    container="body"
-   outsideClick="true">
+   outsideClick="true"
+   i18n-title
+   title="Recent Notifications">
   <i class="fa fa-bell"></i>
```

Comment:
We should add `fa-fw` to all the icons on the navbar and remove the `margin-right`.
ATM they don't have the same size and spacing.